strict.js

```diff
@@ -0,0 +1,57 @@
+module.exports = {
+  "extends": ["./index"],
+  "plugins": [
+    "import",
+    "react-hooks",
+    "react"
+  ],
+  "rules": {
+    "comma-dangle": ["error", { "arrays": "always-multiline", "objects": "always-multiline", "imports": "always-multiline", "exports": "always-multiline" }],
```

Comment:
:joy: good changes here IMO. Especially since we have some apps that aren't being built with ES2017 yet I believe.